pkg/sink/codec/open/open_protocol_decoder.go

```diff
@@ -224,8 +276,23 @@ func (b *BatchDecoder) decodeNextKey() error {
 }
 
 // NewBatchDecoder creates a new BatchDecoder.
-func NewBatchDecoder() codec.RowEventDecoder {
-	return &BatchDecoder{}
+func NewBatchDecoder(ctx context.Context, replicaConfig *config.ReplicaConfig) (codec.RowEventDecoder, error) {
+	var (
+		storage storage.ExternalStorage
+		err     error
+	)
+	if replicaConfig.Sink.KafkaConfig.LargeMessageHandle.EnableClaimCheck() {
+		storageURI := replicaConfig.Sink.KafkaConfig.LargeMessageHandle.ClaimCheckStorageURI
+		storage, err = util.GetExternalStorageFromURI(ctx, storageURI)
+		if err != nil {
+			return nil, cerror.WrapError(cerror.ErrKafkaInvalidConfig, err)
+		}
+		log.Info("claim-check enabled, set the external storage for the kafka consumer",
+			zap.String("storageURI", storageURI))
```

Comment:
if the `storageURI` contains we should mask it.

Reply:
this is using for the internal testing purpose